recipes/azure-storage-cpplite/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "azure-storage-cpplite" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 25f34354fb0400ffe1b5a5c09c793c9fc8104d375910f6c84ab10fa50c0059cb
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("azure-storage-cpplite", max_pin="x.x.x") }}
+
+outputs:
+  - name: azure-storage-cpplite
+    script: build-shared.sh  # [unix]
+    script: bld-shared.bat  # [win]
+    build:
+      activate_in_script: True
+      run_exports:
+        - {{ pin_subpackage("azure-storage-cpplite", max_pin="x.x.x") }}
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - libuuid  # [unix]
+        - libcurl
+        - openssl
+    test:
+      commands:
+        - test -f $PREFIX/include/blob/blob_client.h  # [unix]
+        - test -f $PREFIX/lib/libazure-storage-lite${SHLIB_EXT}  # [unix]
+        - test ! -f $PREFIX/lib/libazure-storage-lite.a  # [unix]
+        - if not exist %PREFIX%\\Library\\include\\blob\\blob_client.h exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\bin\\azure-storage-lite.dll exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\lib\\azure-storage-lite.lib exit 1  # [win]
+  - name: azure-storage-cpplite-static
+    script: build-static.sh  # [unix]
+    script: bld-static.bat  # [win]
+    build:
+      activate_in_script: True
+      run_exports:
+        - {{ pin_subpackage("azure-storage-cpplite", max_pin="x.x.x") }}
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - libuuid  # [unix]
+        - libcurl-static
+        - libopenssl-static
+        - {{ pin_subpackage("azure-storage-cpplite", exact=True) }}
```

Somehow the `run_exports` are not applied inside this recipe. I have seen this multiple times.
```suggestion
        - {{ pin_subpackage("azure-storage-cpplite", exact=True) }}
      run:
        - {{ pin_subpackage("azure-storage-cpplite", exact=True) }}
```

---

Yep, it's a known deficiency that I've also run into a couple of times.
ref: https://github.com/conda/conda-build/issues/3478